packages/components/tests/spec/tooltip_spec.js

```diff
@@ -44,6 +44,18 @@ describe('Test tooltip', function() {
       expect(floating.classList.contains('bx--tooltip--shown')).toBe(true);
     });
 
+    it('Should apply focus to a child element upon opening', function() {
+      element.dispatchEvent(new CustomEvent('click', { bubbles: true }));
+      expect(document.activeElement.closest('.bx--tooltip')).not.toBe(null);
+    });
+
+    // @todo currently not working
+    // it('Should remain open upon click within the tooltip', function() {
+    //   floating.classList.add('bx--tooltip--shown');
```

Comment:
@elizabethsjudd Seems that this test case passes in my env. Did you really see it failing?

Reply:
The behavior is not working when I test it in the browser. If it's passing then the test is currently wrong.

Reply:
OK would you try fixing the test...? More detailed (manual) test script would also help (is it about not wanting tooltip to be closed when user clicks on tooltip body?). Thanks!

Reply:
yes the behavior we want is that when you click on the tooltip body the tooltip remains open.  I'll try updating the test today.